### PR TITLE
[SPINECLI-1190] Add link to postman collection in narrative

### PR DESCRIPTION
**Pull request by axelkrastek1-nhs**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/record-locator/producer.yaml [hunk 1]
```diff
@@ -1219,7 +1221,7 @@ components:
             - DocumentReference
         id:
           type: string
-          pattern: "[A-Za-z0-9\\-\\.]{1,64}"
+          pattern: "^(?=.{1,64}$)[A-Za-z0-9\\.]+-[A-Za-z0-9]+[A-Za-z0-9\\_\\-]*$"
```
Comment (andrewrae5-nhs) [inline specification/record-locator/producer.yaml:1224]: this doesn't look like it's related to the ticket?

**Review by andrewrae5-nhs — APPROVED** [review]
looks fine, just a quick clarification needed

File: specification/record-locator/producer.yaml (hunk 1 above)
Comment (axelkrastek1-nhs) [reply specification/record-locator/producer.yaml]: Reverted this change and opened https://nhsd-jira.digital.nhs.uk/browse/SPINECLI-1599

**Review by andrewrae5-nhs — APPROVED** [review]
looks good!

**Pull request merged**